lighthouse-core/audits/seo/is-crawlable.js

```diff
@@ -122,10 +122,14 @@ class IsCrawlable extends Audit {
           const robotsTxt = robotsParser(robotsFileUrl.href, artifacts.RobotsTxt.content);
 
           if (!robotsTxt.isAllowed(mainResource.url)) {
+            const line = robotsTxt.getMatchingLineNumber(mainResource.url);
             blockingDirectives.push({
               source: {
-                type: /** @type {'url'} */ ('url'),
-                value: robotsFileUrl.href,
+                type: /** @type {'source-location'} */ ('source-location'),
+                url: robotsFileUrl.href,
+                urlProvider: /** @type {'network'} */ ('network'),
+                line: line !== undefined ? line - 1 : 0,
```

Comment:
```suggestion
                line: typeof line === 'number' ? line - 1 : 0,
```

super nit, just a tad clearer IMO

Reply:
wb `const line = robotsTxt.getMatchingLineNumber(mainResource.url) || 1;` and `line: line - 1`

Reply:
wfm 👍